company-profiles/remote-garage.md

```diff
@@ -0,0 +1,31 @@
+# Remote Garage
+
+## Company blurb
+
+[Remote Garage](http://www.remotegarage.club/) is on a mission to bring your best potential regardless of what was left undiscovered, by becoming the first school in India to launch a new career. Sound like a worthy endeavor? Come join us!
```

Comment:
```suggestion
[Remote Garage](http://www.remotegarage.club/) is on a mission to bring out your full potential as a full-stack Go programmer. Sound like a worthy endeavor? Come join us!
```

This change, or something like that, is a much clearer way to state the purpose of Remote Garage (I am guessing a bit at the "full-stack Go programmer" part). You certainly don't have to accept this specific suggestion but there are a couple of problems with the grammar and precision of the current wording:

>on a mission to bring your best potential
"bring out your full potential" is a more natural/correct way to say this.

>regardless of what was left undiscovered,
what would be left "undiscovered"? by whom?

>by becoming the first school in India to launch a new career
which new career? can you prove that you are the first in all of India?